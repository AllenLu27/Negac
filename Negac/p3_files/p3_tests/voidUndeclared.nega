void v;
void fn(){
	int c;
	c = v;
}
