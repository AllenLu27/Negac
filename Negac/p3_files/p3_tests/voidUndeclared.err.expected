FATAL [1,6]: Invalid type in declaration
FATAL [4,6]: Undeclared identifier
Semantic analysis failed
